site/src/components/EditRolesButton/EditRolesButton.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import { ComponentMeta, Story } from "@storybook/react"
+import {
+  MockOwnerRole,
+  MockSiteRoles,
+  MockUserAdminRole,
+} from "testHelpers/entities"
+import { EditRolesButtonProps, EditRolesButton } from "./EditRolesButton"
+
+export default {
+  title: "components/EditRolesButton",
+  component: EditRolesButton,
```

Comment:
if you want, you can add `argTypes: { defaultIsOpen: { defaultValue: true }, ... }` to set prop values that should be used by multiple stories.

Reply:
Way better. Thank you!